nemo/utils/exp_manager.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+import sys
+import time
+from pathlib import Path
+from shutil import copyfile
+from typing import List, Optional, Union
+
+from hydra.utils import get_original_cwd
+from pytorch_lightning.callbacks import ModelCheckpoint
+from pytorch_lightning.loggers import TensorBoardLogger
+
+from nemo.constants import NEMO_ENV_VARNAME_DATETIME
+from nemo.utils import logging
+from nemo.utils.get_rank import is_global_rank_zero
+
+
+def exp_manager(
```

Comment:
This should be refactored to take in all of cfg, and to resolve `logger` from it (and simply do nothing if `logger` namespace doesnt exist). We can use nemo.logging to note that exp_manager has not been used since `logger` is not provided inside config.

Reply:
Sure I can try to resolve it from the config tomorrow.

Reply:
Added a `exp_manager` namespace to configs